src/App.js

```diff
@@ -0,0 +1,16 @@
+import Header from './components/Header';
+import './css/App.css';
+import Books from './pages/Books';
+
+function App() {
+  return (
+    <div className="App d-flex flex-column vh-100">
+      <Header />
+      <main className="main flex-grow-1">
+        <Books />
+      </main>
+    </div>
```

Comment:
- [ ] Please from the project requirement, you should [Setup your router](https://reactrouter.com/en/main/start/tutorial#adding-a-router) inside of the `<App>` component and add 2 routes in your application, each should render a page component:
`/`
`/categories`

  This is how you can setup the router in the app component :point_down: 
  ```jsx
  import React from "react"
  import { BrowserRouter, Routes, Route, Link } from "react-router-dom"
  
  function App() {
    return (
      <BrowserRouter>
        <Navbar />
        <Routes>
          <Route path="/" element={<Books />} />
          <Route path="/categories" element={<Categories />} />
        </Routes>
      </BrowserRouter>
    )
  }
  ```